libs/libcommon/src/libcommon/simple_cache.py

```diff
@@ -356,6 +356,26 @@ def get_cache_reports(kind: str, cursor: Optional[str], limit: int) -> CacheRepo
     }
 
 
+class CacheInfo(TypedDict):
```

Comment:
It already exists as SplitFullName: https://github.com/huggingface/datasets-server/blob/main/libs/libcommon/src/libcommon/simple_cache.py#L58

Let's reuse it

Reply:
Thanks!